src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,34 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class FirstException extends Exception {
+ *       private final int finalErrorCode; //OK
+ *       private int errorCode = 1; // violation needs to be final
+ *       public FirstException() {
+ *           finalErrorCode = 1;
+ *       }
+ *
+ *  }
+ *
+ *   public class FirstError extends Throwable {
+ *       private int errorCode; // violation needs to be final
+ *   }
+ *
+ *   public class FirstClassNotExtendError {
+ *      private int errorCode;  // OK Does not extend any class
+ *      public FirstClassNotExtendError() {
+ *           errorCode = 1;
+ *       }
+ *   }
+ *
+ *   class CustomException extends java.lang.Exception {} // OK no member variable
+ *
+ *   class SecondMutableException extends java.lang.Exception {
+ *       int errorCode; // violation needs to be final
+ *   }
```

Comment:
please fix code indentation. it's not consistent

Reply:
Done

Reply:
there are still problems with indentation.
example:
![image](https://user-images.githubusercontent.com/3431491/164702145-70c56482-6fe2-4f65-9768-d3104898a3ba.png)